salt/cloud/clouds/vmware.py

```diff
@@ -154,12 +155,16 @@ def _get_si():
     password = config.get_cloud_config_value(
         'password', get_configured_provider(), __opts__, search_global=False
     )
+    port = config.get_cloud_config_value(
+        'port', get_configured_provider(), __opts__, search_global=False
```

Comment:
Specify ``default=443`` since that's the default port to be used. If you don't specify a default, authentication will fail for users who don't have the port specified in their providers config